mods/ts/rules/defaults.yaml

```diff
@@ -860,10 +860,14 @@
 		Voice: Move
 	Aircraft:
 		AirborneCondition: airborne
+		CruisingCondition: cruising
+		CruiseAltitude: 4c704
+		AltitudeVelocity: 96
 		LandWhenIdle: false
 		Voice: Move
 		IdealSeparation: 853
 		MaximumPitch: 120
+		FlightDynamics: VTOL, TakeOffOnCreation, TakeOffOnResupply, VTOLTurnToDock
```

Comment:
Aren't Orca fighters and Harpies supposed to be able to hover, while Orca bombers and Banshees cannot? This is probably true for `slide` as well.

Reply:
Omitting `Slide` was intentional, because this is closer to how they moved in the original.

Fixed the missing Hover for Orca/Harpy.